doc/CONFIGURATION.md

```diff
@@ -62,8 +62,11 @@ Define your own e.g.:
 ORDER:
     - 'DP-2'
     - 'Monitor Maker ABC123'
+    - '!^my_regex_here[0-9]+'
 ```
 
+Note that any item prefixed with an `!` will be interpreted as extended POSIX regex, allowing for one to easily create generic rules (e.g. "!^DP-", which will often be sufficient to put external monitors at the top of a column).
```

Comment:
We do need to specify that regex order is in quotes, otherwise the YAML parser will not read it: https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#gotchas

Alternatively, we could use a different character. Looking around for prior art...

Reply:
Given that regex often includes special characters (including '!'), it seems safer to *always* require people to quote their regex rather than encounter surprising bugs later on. I've added a comment to that effect, as requested. If you disagree, feel free to replace this without your favourite character :).